### PR TITLE
Typography and list

### PR DESCRIPTION
**Pull request by amirnathani9**
## change logs
- added spaced list in the list component

## screenshot

image

### CODE REVIEW

File: components/list/list.html [hunk 1]
```diff
@@ -80,6 +81,40 @@ <h3 class="font-bold font-size-6">Basic List</h3>
...
+                        <li class="list-item-container spaced-list">
+                            <span>List Item 2</span>
+                        </li>
+                        <li class="list-item-container spaced-list">
+                            <span>List Item 3</span>
+                        </li>
+                    </div>
+                    <!-- code-snippet  -->
+                    <pre id="myInput" class="relative scroll-style"><code class="language-html">
+        &lt;li class=&quot;list-item-container spaced-list&quot;&gt;
```
Comment (Chiragmodi01) [inline components/list/list.html:104]: I personally used script to change '<' to '&lt'. That way your html structure will look neat.

**Review by harshitpaliwal95 — APPROVED** [review]
good to go

**Review by Chiragmodi01 — APPROVED** [review]
Nothing much to add, code is pretty small.
Overall looks good to me.

File: components/list/list.html (hunk 1 above)
Comment (amirnathani9) [reply components/list/list.html:104]: ok I will look into it

**Pull request merged**